libs/tasks/data-access/src/lib/+state/tasks/tasks.reducer.ts

```diff
@@ -2,17 +2,17 @@ import { createReducer, on, Action } from '@ngrx/store';
 import { EntityState, EntityAdapter, createEntityAdapter } from '@ngrx/entity';
 
 import * as TasksActions from './tasks.actions';
-import { Task } from '../../../../../resources/models/task';
+import { Task } from '../../resources/models/task';
 
 export const TASKS_FEATURE_KEY = 'tasks';
 
 export interface TasksState extends EntityState<Task> {
   tasks: Task[];
-  selectedId?: string | number; // which Tasks record has been selected
+  selectedId: string | null; // which Tasks record has been selected
   loaded: boolean; // has the Tasks list been loaded
-  error?: string | null; // last known error (if any)
-  from?: number;
-  limit?: number;
+  error: string | null; // last known error (if any)
```

Comment:
probably this should be `HttpErrorResponse`

Reply:
jak zmienię zaciąganie danych na poprzez http to podmienię.